### DISCUSSION TITLE
How to write algorithms in typst?

### DISCUSSION THREAD

**Discussion by Madhav-Kanda** [typst/typst#1751] [Q&A] [answered]
I wish to write something like this in typst:

<img width="523" alt="image" src="https://github.com/typst/typst/assets/76394914/e6ae5c7c-c63c-4cb8-af99-2c0a63dfee7d">

Is it possible to do so? If yes, how can I do it?

**Comment by shoukei** [comment, marked as answer]
Would this help?
Have a look at https://github.com/ufodauge/typst-algomod

**Comment by Madhav-Kanda** [comment]
Yes, thank you!
